terraspace/app/stacks/peer-kubernetes-components/example/auth.tf

```diff
@@ -0,0 +1,65 @@
+resource "aws_iam_policy" "config_peer_s3_bucket_policy_read" {
```

Comment:
Do we need an `example` directory under the `stack`?  The stack is the actual use case, so I don't see the point of an `example` in this context.

Reply:
Removed